index.js

```diff
@@ -2,12 +2,25 @@
 const { ProvidePlugin } = require("webpack")
 
 module.exports = class NodePolyfillPlugin {
+	constructor(options = {}) {
+		this.options = options
```

Comment:
We can use object spread syntax to simplify this

Reply:
Like this ?
```js
constructor(options = {}) {
  this.options = { exclude: [], ...options}
}
```

Reply:
👍🏻